application/controllers/logic.py

```diff
@@ -8,38 +8,54 @@
 
 
 def deal_hands(age, players):
+    """Call to have the next age's hands dealt"""
+    if (age not in range(1, 4)) or (len(players) not in range(3, 8)):
```

Comment:
It shouldn't do now. After I wrote this, I made sure some edge cases were accounted for elsewhere in the code.